_posts/2021-09-18-filename-rename-in-pass.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: "CVE-2020-28086 information leakage through third party service in pass"
+author: capitol
+category: security
+---
+![pass-passwordstore](/images/pass-passwordstore.png)
+
+[`pass`](https://www.passwordstore.org/) is a password manager that encrypts your or your
+team's passwords with GPG and those can be stored in a Git repository.
+
+Each password is encrypted in its own file, often named after the service and username.
+For example the password for the user `richard` on the service Spotify might be named
+`spotify/richard.gpg`.
+
+## Trust Boundaries
+
+It's possible to configure your usage of `pass` in many ways, but if you are using it in a team then
+this is a common setup:
+
+Each member has a local machine with a GPG configuration that trusts all other members of the team.
+
+The password store is in Git, and there is a central server to which all team members push and pull.
+
+Here we have three zones:
+
+ * Your own machine
+ * The central server
+ * The other team members
+
+Any operator in one of the zones shouldn't be able to cross a zone barrier and gain
+information or resources from another zone.
+
+## The Attack
+
+If an attacker controls the central Git server or one of the other members' machines, and
+also controls one of the services already in the password store they can do the following:
```

Comment:
It sounds like the service is within the pass store, which put me slightly off.

Reply:
fixed